node/engine/store/durablestore.go

```diff
@@ -210,6 +217,28 @@ func (ds *DurableStore) SetObjective(obj protocols.Objective) error {
 	return nil
 }
 
+// GetLastBlockSeen retrieves the last blockchain block processed by this node
+func (ds *DurableStore) GetLastBlockSeen() (uint64, error) {
```

Comment:
```suggestion
func (ds *DurableStore) GetLastSeenBlockNum() (uint64, error) {
```
This might be nitpicking but I think we should use the term `BlockNum` instead of `Block`, since we're just returning a block number, not an actual block.

Reply:
Updated in https://github.com/statechannels/go-nitro/pull/1581/commits/1185ea1f6ba21612424ee2678b86ee7a4bff284e